qiskit/ignis/verification/tomography/data.py

```diff
@@ -98,17 +101,36 @@ def helper(x, y):
     return ret
 
 
-def count_keys(num_qubits):
-    """
-    Return ordered count keys.
+def count_keys(num_qubits: int) -> List[str]:
+    """Return ordered count keys.
+
+    Args:
+        num_qubits: The number of qubits in the generated list.
+    Returns:
+        The strings of all 0/1 combinations of the given number of qubits
+    Example:
+        >>> count_keys(3)
```

Comment:
this could be made into a code-block too

Reply:
This is redundant for Python code, which is nicely formatted automatically when copied like this from the Python console.